dietpi/dietpi-software

```diff
@@ -3845,6 +3821,32 @@ _EOF_
 
 			Banner_Installing
 
+			# Jessie: https://github.com/Fourdee/DietPi/issues/1620#issuecomment-373086888
+			if (( $G_DISTRO == 3 )); then
+
+				INSTALL_URL_ADDRESS='https://dietpi.com/downloads/binaries/all/libssl1.1_1.1.0f-3+deb9u1_'
+
+				# - ARMv7
+				if (( $G_HW_ARCH < 3 )); then
+
+					INSTALL_URL_ADDRESS+='armhf.deb'
```

Comment:
@Fourdee 
Ah forgot to mention this. In case of shairport sync we didn't install anything on ARMv6, while we didn't disable the install option for this arch. So this must have failed. In case of ARMv6 RPi AFAIK the armhf packages work just fine, but I am not sure about other SBCs? Although debian does not ship specific armv6 binaries, just armv5 (armel), so this must work.

Reply:
@MichaIng 

In terms of Debian/Raspbian, I believe armhf is ARMv6 compiled and is backwards compatible with ARMv7.

Whereas aarch64 is ARMv8 only.

Reply:
@Fourdee 
Okay this is great. So use `armhf` for `armv6l`, if we install via APT repo or deb packages manually pulled from repo, but compile `armv6l` package, if we do this ourself. 👍